examples/main.py

```diff
@@ -0,0 +1,52 @@
+from loggingpy.log import Logger
+import loggingpy.sink as sink
+import time
+import random
+
+
+class InvalidOperationException(Exception):
+    pass
+
+
+if __name__ == "__main__":
+    # Sumologic
+    sumoUri = "https://collectors.de.sumologic.com/receiver/v1/http/ZaVnC4dhaV2bjNoP4SPTO9kxcjdNBNYMhf8MTZci1zG3HYsjnWh4vNCneVeWp7HZGW4cbimAzXDMQLnYTDTfQV9IxAZgLuLcwQD4EA4lWeCk7Ovu1GQpcw=="
```

Comment:
I'm not convinced that exposing our logging tokens on GitHub is such a good idea :)
Can you either purge this from the commit history, or make sure the tokens are invalidated (and inform the team about the new endpoints)?

Reply:
True, I can purge this 🗡